src/react/databrowser/objects/details/Properties.tsx

```diff
@@ -121,13 +122,29 @@ function Properties({ objectMetadata }: Props) {
                 <T.Row>
                   <T.Key> Temperature </T.Key>
                   <T.Value>
-                    {location.isCold ? (
-                      <>
-                        <ColdStorageIcon />
-                        Cold
-                      </>
-                    ) : (
-                      ''
+                    <ColdStorageIcon /> Cold{' '}
+                    {objectMetadata.restore?.ongoingRequest && (
+                      <span>
+                        <i className="fas fa-arrow-alt-circle-up" />
+                        Restoration in progres...
+                      </span>
+                    )}
+                    {objectMetadata.restore?.expiryDate && (
+                      <span>
+                        <i className="fas fa-stopwatch" />
+                        Restored{' '}
+                        <SecondaryText>
+                          (
+                          {Math.floor(
+                            DateTime.fromISO(
+                              objectMetadata.restore?.expiryDate.toISOString(),
```

Comment:
nit:  this optional chaining shouldn't be necessary as it is handled by above condition
```suggestion
                              objectMetadata.restore.expiryDate.toISOString(),
```